tests/cupy_tests/random_tests/common_distributions.py

```diff
@@ -127,6 +127,23 @@ def _check_ks(
             raise AssertionError(message)
 
 
+uniform_params = [
+    {'low': 1, 'high': 10.0, 'size': (3, 5)},
+    {'low': cupy.array([1, 2]), 'high': 3, 'size': None},
+    {'low': 20, 'high': 20.1, 'size': 1000}
+]
+
+
+class Uniform:
+    target_method = 'uniform'
+
+    def test_uniform(self):
+        print("testing uniform")
+        result = self.generate(self.low, self.high, self.size)
+        assert cupy.all(result >= cupy.asarray(self.low).min())
+        assert cupy.all(result < cupy.asarray(self.high).max())
+
```

Comment:
Could you add `test_uniform_ks` that checks the equality of distributions between numpy and cupy with Two-sample Kolmogorov-Smirnov Test?